hack/godep-save.sh

```diff
@@ -81,6 +81,11 @@ done
 rm -rf vendor/github.com/docker/docker/project/
 
 kube::log::status "Updating BUILD files"
+# Assume that anything imported through godep doesn't need Bazel to build.
+# Prune out any Bazel build files, since these can break the build due to
+# missing dependencies that aren't included by godep.
+find vendor/ -type f \( -name BUILD -o -name BUILD.bazel -o -name WORKSPACE \) \
```

Comment:
Would this logic make more sense as part of `hack/update-bazel.sh`?

Reply:
I thought about that, but I didn't want to unnecessarily recreate these files all the time, just only when we're actually updating `vendor/`

Reply:
My thought is, if we're calling `hack/update-bazel.sh` for some reason, we are specifically saying we want to update the bazel files. Is there a large performance difference between updating without deleting, vs deleting?

I also wonder if there's some weird situation we could get into where a build file is updated in some other way, and starts failing the verify-bazel step because of the way it recreates in a temp folder.

I'm not stuck on it, but part of me thinks if you're we're gonna blow them away at all, just blow them all away every time.

Reply:
the other reason not to do this is that `kazel` prints out every time it updates a file, which it'll do every time these are recreated, even though there will likely be no diff.